docs/GET_STARTED.md

```diff
@@ -7,6 +7,14 @@
     - [Xcode]
 - Enable the [fvm] command.
 - Enable the [melos] command.
+    - If you don't have [yq], please install it. [yq] is used to parse the version of melos in the installation command.
```

Comment:
![imo-badge](https://img.shields.io/badge/review-imo-skyblue.svg) 
I think it might be a little hard to understand.

![ask-badge](https://img.shields.io/badge/review-ask-yellowgreen.svg) 
How about doing it like this?
```suggestion
    - Please install the [yq] command if it is not already installed. The yq command is used to parse the `pubspec.lock` file to extract the version of the melos command.
```

Reply:
見返すと確かに分かりにくい表現になっていました。
suggestion使わせていただきました ありがとうございます。